src/public/publicAPIs.ts

```diff
@@ -230,15 +230,27 @@ export function registerAppButtonClickHandler(handler: () => void): void {
 }
 
 /**
- * Registers a handler for hovering the app button.
+ * Registers a handler for entering hover of the app button.
  * Only one handler can be registered at a time. A subsequent registration replaces an existing registration.
- * @param handler The handler to invoke when the personal app button is hovered in the app bar.
+ * @param handler The handler to invoke when entering hover of the personal app button in the app bar.
  */
-export function registerAppButtonHoverHandler(handler: () => void): void {
+export function registerAppButtonHoverEnterHandler(handler: () => void): void {
   ensureInitialized();
```

Comment:
I think we should limit the frame contexts from which this API can be called since it probably won't work as expected everywhere? I feel like this should only be supported in the `FrameContexts.content` for starters.

Reply:
This should be applied to the click handler as well correct?